### PR TITLE
 archived spo2 field to archived_spo2

### LINKED ISSUE #2334: Archive field `spo2` in favour of `ventilator_spo2` in the Daily Rounds model
Currently care front-end only writes to the `ventilator_spo2` field and not `spo2` field.

### Task

- [ ] Archive the field: `spo2` to `archived_spo2`.

### PR DESCRIPTION
**Pull request by praffq**
## Proposed Changes

renamed spo2 to 

### Associated Issue

- Fixes #2334

### Architecture changes

- Remove this section if not used

## Merge Checklist

- [x] Tests added/fixed
- [x] Linting Complete

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@ohcnetwork/care-backend-maintainers @ohcnetwork/care-backend-admins

### CODE REVIEW

**Comment by praffq** [comment]
@rithviknishad  Fixed the migrations issue. Please check

File: care/abdm/utils/fhir.py [hunk 1]
```diff
@@ -190,12 +190,12 @@ def _procedure(self, procedure):
...
-            else None,
-            performedString=f"Every {procedure['frequency']}"
-            if procedure["repetitive"]
-            else None,
+            performedDateTime=(
+                f"{procedure['time']}:00+05:30" if not procedure["repetitive"] else None
+            ),
+            performedString=(
+                f"Every {procedure['frequency']}" if procedure["repetitive"] else None
+            ),
```
Comment (rithviknishad) [inline care/abdm/utils/fhir.py]: @DraKen0009 could you review the files changed in the PR? I think these changes were not intended to be present in this PR.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: care/abdm/utils/fhir.py (hunk 1 above)
Comment (praffq) [reply care/abdm/utils/fhir.py]: @rithviknishad  this file updated by the pre-commits formatting.  I had to update a reference of `spo2` to `archieved_spo2` at line 307 in the file after which this file got formatted as well

**Review by rithviknishad — APPROVED** [review]

File: care/abdm/utils/fhir.py [hunk 2]
```diff
@@ -304,7 +314,7 @@ def _observations_from_daily_round(self, daily_round):
             ),
             self._observation(
                 "SpO2",
-                {"value": daily_round.spo2, "unit": "%"},
+                {"value": daily_round.archived_spo2, "unit": "%"},
```
Comment (sainak) [inline care/abdm/utils/fhir.py]:
```suggestion
                {"value": daily_round.ventilator_spo2, "unit": "%"},
```

we should use ventilator_spo2 in place of the unused field

cc: @khavinshankar

File: care/facility/management/commands/load_event_types.py [hunk 3]
```diff
@@ -116,7 +116,7 @@ class Command(BaseCommand):
                     "name": "VITALS",
                     "children": (
                         {"name": "TEMPERATURE", "fields": ("temperature",)},
-                        {"name": "SPO2", "fields": ("spo2",)},
+                        {"name": "SPO2", "fields": ("archived_spo2",)},
```
Comment (sainak) [inline care/facility/management/commands/load_event_types.py]:
```suggestion
```

we can drop this category, add it to the inactive types list

**Review by sainak — CHANGES_REQUESTED** [review]

File: care/abdm/utils/fhir.py (hunk 2 above)
Comment (khavinshankar) [reply care/abdm/utils/fhir.py]: Yes 👍

**Review by sainak — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**